File: programmers-source/programmers-level1/src/main/java/MidString.java
package main.java;

class MidString {
    public String solution(String s) {
        String answer = "";

        int size = s.length();
        int n = size/2;

        if(size % 2 == 0)
            answer = s.substring(n-1,n+1);
        else
            answer = s.substring(n,n+1);

        System.out.println(answer);
        return answer;
    }

    String getMiddle(String word){
        return word.substring((word.length()-1) / 2, word.length()/2 + 1);
    }

    public static void main(String[] args) {
        MidString sol = new MidString();
        System.out.println(sol.getMiddle("abcde"));
        System.out.println(sol.getMiddle("qwer"));
    }

}

File: Kotlin/kotlin-sample-source/Level1/src/main/kotlin/level1/MidString.kt
package level1

class MidString {
    fun solution(s: String): String {
        var len = s.length / 2

        return if (s.length % 2 == 0)
            s.substring(len-1, len+1)
        else
            s.substring(len, len+1)
    }
}